wsagent/che-core-api-languageserver/src/main/java/org/eclipse/che/api/languageserver/service/WorkspaceService.java

```diff
@@ -162,7 +162,7 @@ public boolean handleResult(
                 InitializedLanguageServer element, List<? extends SymbolInformation> locations) {
               locations.forEach(
                   o -> {
-                    o.getLocation().setUri(removePrefixUri(o.getLocation().getUri()));
+                    o.getLocation().setUri(removeFilePrefix(o.getLocation().getUri()));
```

Comment:
`setUri` method suppose to set uri but here we rather set relative path.

Reply:
I must have misunderstood removePrefixUri, it looks like it is trying to remove "file:///projects" from the beginning of the response but I did not find any cases in which this appeared. My understanding was that removing "file:/projects" would not matter because it was supposed to be removing "file:///projects" to begin with? In which case the original code would have produced a relative path.

Reply:
I see.

Reply:
You don't need to do this, IMO. Just remove this code and the utility method